GG/src/Menu.cpp

```diff
@@ -45,10 +45,10 @@ MenuItem::MenuItem() :
     MenuItem("", false, false)
 {}
 
-MenuItem::MenuItem(bool separator) :
+MenuItem::MenuItem(bool separator_) :
     disabled(true),
     checked(false),
-    separator(true),
+    separator(separator_),
```

Comment:
This is changing behaviour. The parameter is currently ignored.

Reply:
Dropped this change